### PR TITLE
Add test for DDL "set tiflash mode"

### PR DESCRIPTION
**Pull request by hongyunyan**
### What problem does this PR solve?

Issue Number: ref #1175 https://github.com/pingcap/tidb/issues/35850

### What is changed and how it works?

This PR want to add an integration test to make sure "alter table xx set tiflash mode xx" works with the tools.

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### Release note

- No release note

### CODE REVIEW

**Comment by hongyunyan** [comment]
cc @breezewish @lichunzhu

File: pkg/loader/load.go [hunk 1]
```diff
@@ -1008,3 +1008,28 @@ func isSetTiFlashReplica(sql string) bool {
 
 	return false
 }
+
+func isSetTiFlashMode(sql string) bool {
```
Comment (lichunzhu) [inline pkg/loader/load.go:1012]: Can we merge this function into `isSetTiFlashReplica`? They look so similar except for L1029

Comment (hongyunyan) [reply pkg/loader/load.go]: Yes, I think we can merge the function.

File: pkg/loader/load.go [hunk 2]
```diff
@@ -452,7 +452,7 @@ func (s *loaderImpl) processMysqlDDL(ddl *DDL) error {
 		return nil
 	})
 
-	if err != nil && isSetTiFlashReplica(ddl.SQL) {
+	if err != nil && isSetTiFlashReplicaOrTiFlashMode(ddl.SQL) {
```
Comment (breezewish) [inline pkg/loader/load.go:455]: Maybe we should ignore all errors related with TiFlash? Since TiFlash statements are not available in other dbs.

**Review by breezewish — APPROVED** [review]

**Pull request merged**